grails-app/views/projectDocument/_index.gsp

```diff
@@ -0,0 +1,26 @@
+<g:javascript>
+    const user = {
+        <g:if test="${session?.user}">
+            displayName: "${session.user.displayName}",
+            email: "${session.user.emailAddress}",
+            userName: "${session.user.userName}"
+        </g:if>
+    };
+    const component = {
+      user: user,
+      projectKey:'${issue.projectKey}',
+      attachedDocumentsUrl: "${createLink(uri: '/api/files-helper/attached-documents', method: 'GET')}",
+      attachDocumentsUrl: "${createLink(uri: '/api/files-helper/attach-document', method: 'POST')}",
+      issue: '${issue.requestDate}',
```

Comment:
This could be derived from a call to get project by project key, which is above.